src/OrchardCore.Modules/OrchardCore.Media/Controllers/AdminController.cs

```diff
@@ -10,9 +10,8 @@
 using Microsoft.Extensions.Localization;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
-using OrchardCore.DisplayManagement.Notify;
 using OrchardCore.FileStorage;
-using OrchardCore.Media.Services;
```

Comment:
Don't change the namespace of this attribute in this case, and you can even preserve the folder it was in when you move it to the Core project.